modules/statics/src/errors.ts

```diff
@@ -26,6 +26,13 @@ export class DuplicateCoinDefinitionError extends BitGoStaticsError {
   }
 }
 
+export class DuplicateCoinIdDefinitionError extends BitGoStaticsError {
```

Comment:
Can you kindly add a test case for this error flow?

Reply:
done